DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/DelegateCourseInfoViewModel.cs

```diff
@@ -0,0 +1,74 @@
+namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates
+{
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Data.Models.Courses;
+    using DigitalLearningSolutions.Data.Models.CustomPrompts;
+
+    public class DelegateCourseInfoViewModel
+    {
+        public DelegateCourseInfoViewModel(
+            DelegateCourseInfo info,
+            List<CustomPrompt>? customPrompts,
+            (int totalAttempts, int attemptsPassed) attemptStats
+        )
+        {
+            CustomisationId = info.CustomisationId;
+            ApplicationName = info.ApplicationName;
+            CustomisationName = info.CustomisationName;
+            Supervisor = info.Supervisor;
+            Enrolled = info.Enrolled.ToString("dd/MM/yyyy");
+            LastUpdated = info.LastUpdated.ToString("dd/MM/yyyy");
+            if (info.CompleteBy.HasValue)
+            {
+                CompleteBy = info.CompleteBy.Value.ToString("dd/MM/yyyy");
+            }

```

Comment:
Could these be simplified to the following:
CompleteBy = info.CompleteBy?.ToString("dd/MM/yyyy"); 

Also you might be able to get away with .ToShortDateString().

Reply:
Oh yeah.

Can't do ToShortDateString because that depends on locale (e.g. fails on github runner because it uses MM/dd/yyyy format).